build_runner/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.8.2
 
+- Allow passing multiple `--output` options. Each option will be split on
+  `:`. The first value will be the output directory, the second value will
```

Comment:
I thought we were going to do the reverse of this? `<input-dir>:<output-dir>`

Reply:
Replied above.